java/src/jmri/jmrit/symbolicprog/tabbedframe/PaneProgPane.java

```diff
@@ -2620,7 +2620,7 @@ public void printPane(HardcopyWriter w) {
                 // Java 1.5 has a known bug, #6328248, that prevents printing of progress
                 //  bars using old style printing classes.  It results in blank bars on Windows,
                 //  but hangs Macs. The version check is a workaround.
-                float v = Float.valueOf(java.lang.System.getProperty("java.version").substring(0, 3));
+                float v = Float.parseFloat(System.getProperty("java.version").substring(0, 3));
                 if (originalName.equals("Speed Table") && v < 1.5) {
```

Comment:
We doesn't support Java 1.5 anymore so I think this section could be removed.
But it's probably better to do that in a separate PR.